apps/examples/st_things/Kconfig

```diff
@@ -44,11 +44,17 @@ config EXAMPLES_ST_THINGS_BLINK
 	---help---
 		This is a Blink Sample.
 
+config EXAMPLES_ST_THINGS_MSG_HANDLING
+	bool "Message Handling Example"
+	select IOTBUS_GPIO
+	---help---
+		This is a Blink Sample.
```

Comment:
Please change this descript

Reply:
If you want to use RESET funtion here, you need to set depends on xxxx at RESET_BUTTON